rust/origen/src/core/model/pins.rs

```diff
@@ -657,6 +658,8 @@ impl StateTracker {
                     pins.insert(
                         phys.name.clone(),
                         vec![(
+                            // unsure if this could be a group or guaranteed to be individual pin
+                            phys.name.clone(),
```

Comment:
@coreyeng the <code>Pin</code> struct is always individual pins, correct?

Reply:
A `Pin` is, yes. Its always a representative of a single, _physical_ pin and should always be unique.